protocol/x/subaccounts/keeper/isolated_subaccount.go

```diff
@@ -20,72 +53,88 @@ import (
 // Returns a `successPerUpdates` value, which is a slice of `UpdateResult`.
 // These map to the updates and are used to indicate which of the updates
 // caused a failure, if any.
+// Returns a `isolatedPerpetualStateTransitions` value, which is a slice of
+// `isolatedPerpetualStateTransition`.
+// These map to the updates and are used to indicate which of the updates opened or closed an
+// isolated perpetual position.
 func (k Keeper) checkIsolatedSubaccountConstraints(
```

Comment:
nit: it seems weird that this function is named `checkIsolatedSubaccountConstraints` (focus: `check`) and it mainly calls `processIsolatedPerpetualUpdates`  (focus: `process`) for each `settledUpdate`. 

Should this instead be 
- inner function: `processSingleIsolatedPerpetualUpdate`
- outer function `processIsolatedPerpetualUpdates`

Reply:
Good point, I struggled with renaming, and your proposed function names sound good.